src/absil/il.fs

```diff
@@ -487,7 +487,7 @@ type ILScopeRef =
     | Local
     | Module of ILModuleRef
     | Assembly of ILAssemblyRef
-    | PrimaryAssembly
+    | PrimaryAssembly of ILAssemblyRef option
```

Comment:
I need to look at this tomorrow. My gut tells me we shouldn't have to add a field to ILScopeRef.PrimaryAssembly. Is there no way around this?

Reply:
@KevinRansom we should take some time and chat about it tomorrow